### DISCUSSION TITLE
Does detekt CLI works with formating rules ?

### DISCUSSION THREAD

**Discussion by VovaStelmashchuk** [detekt/detekt#3939] [Q&A] [answered]
We use detekt in our CI, we use detekt CLI. Detekt does not report errors from formatting rules ?
Can i turn on formating rules by CLI ?

**Comment by cortinico** [comment, marked as answer]
>  Can i turn on formating rules by CLI ?

Yes you can. You would have to use the `--plugin` and pass as parameter the `detekt-formatting` Jar you can download from here: https://repo1.maven.org/maven2/io/gitlab/arturbosch/detekt/detekt-formatting/1.17.1/

**Reply by VovaStelmashchuk** [reply]
Thx Does it normal behavior ? May be detekt need to turn on this functional by default. for consistency with other modules

**Reply by cortinico** [reply]
`detekt-formatting` is an external ruleset by design. It wraps KtLint rules and and it's up to you to decide if you want to include it as well or not. More on this here: https://detekt.github.io/detekt/extensions.html

The rationale behind this is because we don't want to tight couple Detekt with KtLint, so users that don't use it can exclude the plugin entirely.
